tsconfig.json

```diff
@@ -35,6 +35,9 @@
     "skipLibCheck": false,
     "strict": false,
     "plugins": [
+      {
+        "name": "@react-spectrum/ts-plugin"
```

Comment:
macro-ts-plugin?
or is this leaving it open to other things we may want to do with the plugin?